docs/source/include/operator.rst

```diff
@@ -111,17 +110,17 @@ Using A LinearOperator
 We implement two ways to evaluate a :class:`.LinearOperator`. The first is using standard
 callable syntax: ``A(x)``. The second mimics the NumPy matrix multiplication
 syntax: ``A @ x``. Both methods perform shape and type checks to validate the
-input before ultimately either calling `A._eval` or generating a new :class:`.LinearOperator`.
+input before ultimately either calling :code:`A._eval` or generating a new :class:`.LinearOperator`.
```

Comment:
consider adding a note about ":code:" to https://scico.readthedocs.io/en/stable/style.html#markup